### PR TITLE
LPS-120231 No option to exit from the Verification site

### PR DESCRIPTION
**Pull request by erickmonteiroo**

### CODE REVIEW

**Comment by erickmonteiroo** [comment]
ci:test:sf

**Comment by erickmonteiroo** [comment]
ci:test:sf

**Comment by erickmonteiroo** [comment]
ci:test:sf

**Comment by erickmonteiroo** [comment]
ci:test:relevant

File: portal-web/docroot/html/portal/update_email_address.jsp [hunk 1]
```diff
@@ -81,6 +81,8 @@ if (referer.equals(themeDisplay.getPathMain() + "/portal/update_email_address"))
 
 			<aui:button-row>
 				<aui:button type="submit" />
+
+				<aui:button href="<%= themeDisplay.getPathMain() %>" type="cancel" />
```
Comment (drewbrokke) [inline portal-web/docroot/html/portal/update_email_address.jsp:85]: @erickmonteiroo does this need to have the `"/portal/logout"` appended to it as well? Or is the user not logged in at this view?

**Review by drewbrokke — COMMENTED** [review]
@erickmonteiroo this seems good, just the one inline comment.

File: portal-web/docroot/html/portal/update_email_address.jsp (hunk 1 above)
Comment (erickmonteiroo) [reply portal-web/docroot/html/portal/update_email_address.jsp:85]: On this one the user is logged in, but doesn't need to log out. The cancel button will redirect to the main page that is a page to confirm the email address.

**Comment by drewbrokke** [comment]
ci:foward

LGTM 👍

**Comment by drewbrokke** [comment]
ci:forward

**Pull request closed without merging**